spec/build-spec.js

```diff
@@ -35,6 +37,8 @@ describe('Build', () => {
         return specHelpers.vouch(fs.realpath, dir);
       }).then( (dir) => {
         directory = dir + '/';
+        const createdHomeDir = temp.mkdirSync('atom-build-spec-home');
+        os.homedir = () => createdHomeDir;
```

Comment:
Since the working directory is generated async, it makes sense to keep it consistent.
Should be easy enough to copy from that.